modules/ROOT/pages/dw-core-functions-max.adoc

```diff
@@ -8,11 +8,11 @@ endif::[]
 [[max1]]
 == max&#40;Array<T&#62;&#41;: T &#124; Null
 
-Returns the highest numeric value in an array.
+Returns the highest `Comparable` value in an array.
 
 
-Returns null if the array is empty and produces an error when non-numeric
-values are in the array.
+The items must be of the same type. If not, the function throws an error, and
+it returns `null` when the array is empty.
```

Comment:
> The items must be of the same type or the function throws an error. The function returns `null` if the array is empty.

Reply:
done in adoc file only, will update dw repo separately

Reply:
done in dw